setup.py

```diff
@@ -5,6 +5,7 @@
 PY_VER = sys.version_info
 install_requires = [
     "aiohttp>=1.0.0",
+    "attrs>=21.4.0,<22.0.0",
```

Comment:
This looks too restricted and might cause inconvenience to end-users. I'd suggest relaxing it a bit from the left side (a few years of releases back) and don't restrict the right side at all, for example: 
```suggestion
    "attrs>=20.1",
```

What do you think?

Reply:
I can agree with the lower bound as user can have some other packages installed that require an earlier version of the lib.

The reasoning behind the upper bound is that we cannot be sure that our code is compatible with a next major version of a dependency.

I suggest putting "attrs>=20.1,<22.0" here, set "attrs==20.1" in `requirements.txt`. But then we will need to manually test with "21.x" as we have no guarantees that our code works with this major version.

Reply:
Also, the same should be true for the `aiohttp` dependency.

Reply:
> we cannot be sure that our code is compatible with a next major version of a dependency.

Yeah, nobody can guarantee future compatibility. With no upper bound restriction, if a user experiences any future compatibility issue, they can restrict the upper bound by themselves - no problem. However, if ansq restricts upper bound attr version and new shiny attr version would appear then the user couldn't use it at all. My point is - let the user decide what to do, not the software.

Reply:
Good point. I agree.

Reply:
Addressed in https://github.com/list-family/ansq/pull/45/commits/abd0e07d32272425c88c2f35a9eefe2d0827ba84

Reply:
Thanks!